plugin-server/src/main/pluginsServer.ts

```diff
@@ -512,7 +519,14 @@ export async function startPluginsServer(
             const hub = await setupHub()
             const postgres = hub?.postgres ?? new PostgresRouter(serverConfig)
             const batchConsumerFactory = new DefaultBatchConsumerFactory(serverConfig)
-            const ingester = new SessionRecordingIngesterV2(serverConfig, true, postgres, batchConsumerFactory)
+            const producer = hub?.kafkaProducer ?? (await KafkaProducerWrapper.create(serverConfig))
+            const ingester = new SessionRecordingIngesterV2(
```

Comment:
More of a small thing but if we could start just using the "hub" instead of passing each var in that would be great. Just means each service uses the same pattern which makes it easier to refactor alltogether later